src/index.scss

```diff
@@ -21,6 +23,10 @@
 		display: none;
 	}
 
+	.components-button {
+		font-size: $baseline-interactive-font-size;
+	}
```

Comment:
Hoist these existing styles to ensure they are applied to buttons within popovers that now render in the root body element.